lambdatest/README.md

```diff
@@ -29,7 +29,7 @@ Here's how you can track incidents in Datadog with LambdaTest:
 2. Select settings from the menu bar.
 3. Click on integrations under settings.
```

Comment:
When a user clicks "Connect Accounts" on the datadog tile, they'll be taken to the onboarding_url that you specified when creating the OAuth client, which I believe is this Lambdatest integrations page. If so, we can remove lines 28-30 and replace with the following:

```suggestion
1. Click "Connect Accounts" to begin authorization of the LambdaTest integration. You will be taken to the Integrations page in LambdaTest.
```